teamC/choojaeho/app.js

```diff
@@ -0,0 +1,43 @@
+const http = require ("http");
+const express = require ("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config()
+
+const {DataSource} = require('typeorm')
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+// myDataSource.initialize()
+//     .then(()=>{
+//         console.log("Data Source has been initialized!")
+//     })
+
+const app=express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan(''));
+
+
+
+app.get("/ping", (req,res) => {
+    res.json({message : "PONG"});
```

Comment:
message만 반환하는것이 아닌 status code 도 항상 같이 반환 될 수 있게 해주세요~~
```suggestion
    res.status(200).json({message : "PONG"});
```